pkg/nsx/services/common/store.go

```diff
@@ -131,12 +143,21 @@ func (service *Service) InitializeResourceStore(wg *sync.WaitGroup, fatalErrors
 	resourceParam := fmt.Sprintf("%s:%s", ResourceType, resourceTypeValue)
 	queryParam := resourceParam + " AND " + tagParam
 
+	if org != "" || project != "" {
+		// QueryClient.List() will escape the path, "path:" then will be "path%25%3A" instead of "path:3A",
+		//"path%25%3A" would fail to get response. Hack it here.
+		path := "\\/orgs\\/" + org + "\\/projects\\/" + project + "\\/*"
```

Comment:
search can under "infra" scope or "org" or "project" scope?
If only org provided, that means search under "org" scope? then we do not need to add /projects/<project-id>

Reply:
Actually, InitializeVPCResourceStore is not used yet, we now only use InitializeResourceStore to query all resources, we can update it in the future if there are scenarios when only query under "org" or "org && project".